client/landing/stepper/declarative-flow/internals/steps-repository/__user/use-create-user-mutation.ts

```diff
@@ -21,6 +24,9 @@ export function useCreateAccountMutation() {
 			} );
 		},
 		onSuccess: ( data ) => {
+			if ( 'isNewAccountCreated' in data && data.isNewAccountCreated ) {
+				setIsNewUser( true );
```

Comment:
One thing that, I think, would be nice to make sure it is clear is that the user will be "newUser"  just once. For instance, if you create the user, then land on the domain's step and refresh the page, the user is not new anymore.

Reply:
Well, we could persist that, but I'm always worried about persistence bugs 🤔

Reply:
> Well, we could persist that, but I'm always worried about persistence bugs 🤔

I think we could have a better definition of what is a "newUser"

Reply:
I replicated what is done during user creation here https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/signup/api/account.tsx#L120 😄